src/extractor/guidance/roundabout_handler.cpp

```diff
@@ -111,32 +111,57 @@ void RoundaboutHandler::invalidateExitAgainstDirection(const NodeID from_nid,
     if (in_edge_data.roundabout || in_edge_data.circular)
         return;
 
-    bool past_roundabout_angle = false;
-    const bool lhs = profile_properties.left_hand_driving;
-    const int step = lhs ? -1 : 1;
-    for (std::size_t cnt = 0, idx = lhs ? intersection.size() - 1 : 0; cnt < intersection.size();
-         ++cnt, idx += step)
+    std::cout << "invalidateExitAgainstDirection\n";
+
+    // Find range in which exits that must be invalidated (shaded areas):
+    // exit..end   exit..end  begin..exit for ↺ roundabouts
+    // ⭦           ⭦  ⭩         ⭦
+    //  ⭡⭧          ⭡⭩ ▒       ▒⭨⭡
+    //  ⭡⭦          ⭡⭨▒▒       ▒▒⭡⭨
+    // ⭧▒▒⭦        ⭧▒▒▒▒      ▒▒⭧
```

Comment:
It looks like a font issue, because boxes are supported by the font. Anyway i changed to the markdeep style, so rendered 
image ![](https://cloud.githubusercontent.com/assets/4421046/25616296/1ad89bd6-2f3c-11e7-9dd1-2225651ce472.png)